docs/tutorials/elasticdl_on_prem_cluster.md

```diff
@@ -69,3 +69,48 @@ elasticdl train \
  --image_pull_policy=Always \
  --namespace=kubemaker
 ```
+
+## Add Cluster-specific Information
+
+If the on-prem cluster requires additional specifications to pods or services,
+such as labels, tolerations, etc, we can add an additional argument
```

Comment:
labels are for specifying fault tolerance-related specifications, it doesn't make sense to put labels and tolerance side-by-side.

Reply:
Revised.